src/styles/card.scss

```diff
@@ -0,0 +1,83 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  font-family: $bodyFont;
+  padding: 32px 16px 16px;
+  height: 406px;
+  transition-duration: 300ms;
+
+  &:hover {
+    transform: scale(120%);
+    color: #34568b;
+  }
+
+  &__image {
```

Comment:
The BEM element `&__image` is missing a `border-radius` property. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), if the design has images with rounded corners, they should be styled accordingly.